quinn/tests/many_connections.rs

```diff
@@ -0,0 +1,197 @@
+#![cfg(feature = "slow_tests")]
```

Comment:
Would [the `#[ignore]` attribute](https://doc.rust-lang.org/book/ch11-02-running-tests.html#ignoring-some-tests-unless-specifically-requested) be a better way to accomplish this? Along with adding `--ignored` to the CI script.

Reply:
I think features are more appropriate for this, why would you prefer ignore?

Reply:
The documentation example is for this exact use-case, and it ensures that the test is still typechecked and built as part of the normal workflow.

Reply:
What documentation example do you mean here? I don't think we have any other tests that really exercise this specific use case (including large payloads).

Plus, I would value making the added dependencies optional conditional on the feature flag.

Reply:
Indeed docs do say that:
```
 Rather than listing as arguments all tests you do want to run, you can instead annotate the time-consuming tests using the ignore attribute to exclude them

When you’re at a point where it makes sense to check the results of the ignored tests and you have time to wait for the results, you can run cargo test -- --ignored instead.
```

The only problem I see with `ignored` is that I will have to decorate helper functions as unused.

Reply:
> What documentation example do you mean here?

The example(s) in the documentation for the ignore attribute: https://doc.rust-lang.org/book/ch11-02-running-tests.html#ignoring-some-tests-unless-specifically-requested

> The only problem I see with ignored is that I will have to decorate helper functions as unused.

`#![allow(unused)]` at the top of the module, perhaps?

Reply:
Wait, why would you have to decorate anything as unused? The function is still compiled and referenced, just not invoked at runtime without a flag; that's still a static use.

Reply:
Good point, updated the PR ;)